tests/test_meshtastic_utils_message_edge.py

```
@@ -1198,3 +1268,730 @@ def capture_debug(*args, **kwargs):
+        """Test check_connection when probe submission raises RuntimeError."""
+        mu.config = {
+            "meshtastic": {
+                CONFIG_KEY_CONNECTION_TYPE: CONNECTION_TYPE_TCP,
+                "health_check": {"enabled": True, "initial_delay": 0},
+            }
+        }
+        mu.meshtastic_client = Mock()
+        mu.reconnecting = False
+        mu.shutting_down = False
+
+        async def run_test():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `run_test`

Add return type annotation: `None`

(ANN202)